harvester/cli.py

```diff
@@ -48,7 +50,7 @@ def harvest(host, from_date, until, format, out, verbose):
     try:
         responses = mysickle.ListIdentifiers(
             **{'metadataPrefix': format,
-               # 'set': 'hdl_1721.1_33972'
+               'set': set,
```

Comment:
Do we know what the behavior is if this is set to `None`?

Reply:
No. But this is very important to make sure it works as it does _before_ this PR when we are done and no set is provide.

Reply:
Just to clarify, I think what I'm suggesting is that we only add the `set` key to the dictionary if it has been provided on the command line. Otherwise, we are explicitly setting this to the value `None` which may impact how the request is handled.

Reply:
The request seems to go through fine if set=None. Should I just do that or add an if-else for whether a set is specified?

Reply:
It would probably be fine, based on what I looked at in the sickle source and observed in requests' behavior. I'd still go ahead and just add an if block to add the `set` key if it's been provided. Leaving it as `None` is essentially undefined behavior and could cause problems in the future.

Reply:
Will do and then I'll merge the commits